metricbeat/module/kubelet/container/_meta/data.json

```diff
@@ -0,0 +1,78 @@
+{
+    "@timestamp": "2017-04-06T15:29:27.150Z",
+    "beat": {
+        "hostname": "beathost",
+        "name": "beathost",
+        "version": "6.0.0-alpha1"
+    },
+    "kubelet": {
+        "container": {
+            "cpu": {
+                "usage": {
+                    "corenanoseconds": 3305756719,
+                    "nanocores": 5992
```

Comment:
@ruflin 
```
// CPUStats contains data about CPU usage.
type CPUStats struct {
	// The time at which these stats were updated.
	Time metav1.Time `json:"time"`
	// Total CPU usage (sum of all cores) averaged over the sample window.
	// The "core" unit can be interpreted as CPU core-nanoseconds per second.
	// +optional
	UsageNanoCores *uint64 `json:"usageNanoCores,omitempty"`
	// Cumulative CPU usage (sum of all cores) since object creation.
	// +optional
	UsageCoreNanoSeconds *uint64 `json:"usageCoreNanoSeconds,omitempty"`
}
```